packages/testing/src/chai-retry-plugin/chai-retry-plugin.ts

```diff
@@ -0,0 +1,109 @@
+import Chai from 'chai';
+import { retryFunctionAndAssertions } from './helpers';
+
+import type { AssertionMethod, FunctionToRetry, AssertionStackItem, RetryOptions, PromiseLikeAssertion } from './types';
+
+const { Assertion } = Chai;
+
+declare global {
```

Comment:
can we move it to the `index.d.ts` file?

Reply:
@yurii-ve moved it to `types.ts` as temporary fix